root/layout/components/BottomMenu.js

```diff
@@ -54,15 +55,17 @@ const LanguageMenu = () => (
       )}
     </span>
     <ul>
-      {$c.stash.server_languages.map(function (l, index) {
```

Comment:
Can we decouple this from $c.stash and instead pass server_languages as a prop?

Reply:
We could, but root/layout/components/Head.js is still used with `React.embed`, so we have to keep `server_languages` in the stash either way.

It would also be slightly trickier than usual, since it's used by *every single page*, so every Catalyst controller action will need it added to `component_props`. We can do that by setting a default `component_props` in the "root" controller, but all the downstream actions will need to be changed to not overwrite `component_props` but to merge values into it.

Reply:
See https://github.com/metabrainz/musicbrainz-server/pull/620/commits/cbca98e3309e0818f04d7a51f92e9bc4d7692020